src/consts/chains.ts

```diff
@@ -47,6 +47,25 @@ export const chains: ChainMap<ChainMetadata & { mailbox?: Address }> = {
     ...chainMetadata.solanadevnet,
     mailbox: '4v25Dz9RccqUrTzmfHzJMsjd1iVoNrWzeJ4o6GYuJrVn',
   },
+  cosmoshub: {
```

Comment:
are we using this?

Reply:
Yeah, we need a chain that all wallets should already have b.c. the cosmos-kit hooks throw errors in certain cases where there are no cosmos chains provided to them (like on `main` branch where there are no cosmos tokens/chains configured)